modules/RTC/RTCBrowserType.js

```diff
@@ -176,14 +176,6 @@ const RTCBrowserType = {
     },
 
     /**
-     * Whether the browser is running on an android device.
-     * @returns {boolean}
```

Comment:
So now chrome on android works without explicitly setting the resolution to 180p and the frame rate to 15?